udata/core/organization/views.py

```diff
@@ -93,6 +98,23 @@ def get_context(self):
         return context
 
 
+@blueprint.route('/<org:org>/rdf', localize=False)
+def rdf(org):
+    '''Root RDF endpoint with content negociation handling'''
+    format = RDF_EXTENSIONS[negociate_content()]
+    url = url_for('organizations.rdf_format', org=org.id, format=format)
+    return redirect(url)
+
+
+@blueprint.route('/<org:org>/rdf.<format>', localize=False)
+def rdf_format(org, format):
```

Comment:
Use the logic from here https://github.com/opendatateam/udata/blob/f84280262dd78b85fb4793d8702c502e6c96a3d8/udata/core/site/views.py#L269 to output the datasets of the org.